package.json

```diff
@@ -34,9 +39,14 @@
     "css-annotation": "^0.6.2",
     "deep-diff": "^0.3.4",
     "dnd-core": "^2.0.2",
+    "express": "^4.14.1",
+    "express-graphql": "^0.6.3",
     "faker": "^3.1.0",
     "fibers": "^1.0.14",
     "font-awesome": "^4.6.3",
+    "graphql": "^0.9.1",
```

Comment:
Graphql dependencies should be in another story right. Keep this branch pure please